integration/apps/nasft/nasft/npbparams.fi

```diff
@@ -41,7 +41,7 @@
 !
         integer nx, ny, nz, maxdim, niter_default
         integer*8 np_min, ntdivnp
-        parameter (nx=2048, ny=1024, nz=1024, maxdim=2048)
+        parameter (nx=4096, ny=2048, nz=2048, maxdim=4096)
```

Comment:
I'm working on a patch that will enable this to be selected, if this is merged then nas-ft will not run on mcfly

Reply:
OK, I'll remove it from this stack.